pkg/rancher-components/src/components/Banner/Banner.vue

```diff
@@ -164,6 +164,7 @@ $icon-size: 24px;
     width: 100%;
     border-left: solid $left-border-size transparent;
     display: flex;
```

Comment:
Is there a reason that we converts these elements into Flex boxes? Considering this is text `inline-block` is usually the standard display type of all the text-like elements.
If we are not able to find out why, let's just approve the fix and review it another time.

Reply:
Yeah, but why would you inject a `paragraph` instead of a `span`? 😬 
That's ABC of the HTML and I would rather flag a PR which does that.

Reply:
Anyhow, since we do not want to make an inquisition about this, I'll rather create an issue for this case and remove all the injected values of this type, unless we can set any type of automation.

Reply:
Added https://github.com/rancher/dashboard/issues/8708